bokeh/tests/test_actions_deprecation.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import
+
+import unittest
+import warnings
+
+from bokeh.models import Action as modelsAction
+from bokeh.models.actions import Callback, OpenURL, Action
+from bokeh.models import callbacks
+
+
+class ActionsDeprecationTests(unittest.TestCase):
```

Comment:
Until we actually get to a point where we offer selectable compatibility modes (`bokeh.set_version("0.11.2")`), and have to e.g., verify bug/behaviour compatibility in different selected versions, I think tests like this are probably more work than useful.